assets/queries/openAPI/property_defining_maximum_not_greater_than_minimum/metadata.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "ab2af219-cd08-4233-b5a1-a788aac88b51",
+  "queryName": "Property Defining Maximum Not Greater Than Minimum",
```

I think it would look better this way:
```suggestion
  "queryName": "Property Defining Minimum Greater Than Maximum",
```